plugins/provisioners/ansible/helpers.rb

```diff
@@ -25,6 +25,12 @@ def self.stringify_ansible_playbook_command(env, command)
         shell_command += shell_arg.join(' ')
       end
 
+      def self.expand_path_in_unix_style(path, base_dir)
```

Comment:
@sethvargo do prefer to make this helper function available to a larger scope of vagrant code base?